.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Dev",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  // 👇 Features to add to the Dev Container. More info: https://containers.dev/implementors/features.
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {},
+    "ghcr.io/xe/devcontainer-features/fish:0.2.0": {
+      "fisher": true
+    },
+    "ghcr.io/devcontainer-community/devcontainer-features/atuin.sh:1": {},
+    "ghcr.io/devcontainers-extra/features/neovim-apt-get:1": {},
+    "ghcr.io/joshuanianji/devcontainer-features/aws-cli-persistence:1": {}
+  },
+  "initializeCommand": "mkdir -p ${localEnv:HOME}${localEnv:USERPROFILE}/.local/share/atuin",
+  "postCreateCommand": "npm ci",
+  "mounts": [
+    {
+      "type": "bind",
+      "source": "${localEnv:HOME}${localEnv:USERPROFILE}/.local/share/atuin",
+      "target": "/home/vscode/.local/share/atuin"
+    }
+  ],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "unifiedjs.vscode-mdx",
```

Comment:
question: do we need these exntesions?

Reply:
At least we need prettier, I copied this list of extensions from the blog's devcontainer manifest.